national-resources/switzerland.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Switzerland
+country_code: CH
+contributors: [Aitana Neves, Patricia M. Palagi] 
+coordinators: [Aitana Neves]
+---
+
+## Introduction 
+<!---General Infectious diseases data considerations for your country--->
+
+Switzerland is a federal republic composed of 26 cantons. Switzerland's health system is highly decentralised, with significant responsibility for public health measures and policies resting with the country's cantons. The [Federal Office of Public Health](https://www.bag.admin.ch/bag/en/home.html) (FOPH) is responsible for coordinating national health policies, but the cantons have a high degree of autonomy in implementing these policies.
+
+
+## Health authorities
+<!--- A section to list and provide context to agencies/authorities/institutions which define public health measures and policies --->
+The FOPH plays a critical role in setting national public health goals and guidelines, overseeing the country's health insurance system, and coordinating responses to public health emergencies. It has a Communicable Diseases Division in charge of monitoring, publishing regular reports on the epidemiological situation, laying down and implementing prevention and control strategies. The FOPH also manages national health registries and monitors health indicators, such as disease incidence rates and vaccination coverage. FOPH may also require laboratories, health care institutions and other institutions of the cantons to communicate their stocks of therapeutic products, protective equipment and other medical goods important for the maintenance of health care capacities (important medical goods). [DigiSanté](https://www.bag.admin.ch/bag/fr/home/strategie-und-politik/nationale-gesundheitsstrategien/digisante.html), a national programme to foster the digital transformation of the healthcare systems and public health authorities, has just launched.
```

Comment:
"has just launched" - can you add a date (otherwise the meaning changes depending when one reads the sentence 2023 vs 2026).

Reply:
Indeed! I have adapted that sentence (and ended up adding more information.